cube/cube.c

```diff
@@ -4492,8 +4516,6 @@ static VkSurfaceFormatKHR pick_surface_format(const VkSurfaceFormatKHR *surfaceF
 static void demo_init_vk_swapchain(struct demo *demo) {
     VkResult U_ASSERT_ONLY err;
 
-    demo_create_surface(demo);
```

Comment:
Move demo_create_surface to time between demo_init and demo_select_physical_device.